src/jamf_pro_sdk/clients/classic_api.py

```diff
@@ -173,6 +178,69 @@ def delete_computer_by_id(self, computer: ComputerId) -> None:
         computer_id = ClassicApi._parse_id(computer)
         self.api_request(method="delete", resource_path=f"computers/id/{computer_id}")
 
+    def set_computer_unmanaged_by_id(self, computer: ComputerId) -> None:
+        """Sets the management status to `unmanaged` for a single computer using the ID
+
+        ..important::
```

Comment:
```suggestion
        .. important::
```
Missing a space after `..`.